.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 .idea
 .env
+specs/kvstore/out/*
```

Comment:
Do we need this?

Reply:
(Or should it be expanded to specs/**/out/* or something like that?

Reply:
I think `specs/**/out/* ` would be better. 

Every time the model runs, it dumps a lot of data like state graph in the `out/` directory for each run. (This is what is used for generating tests and creating debug information.)